bin/mocha

```diff
@@ -16,7 +16,7 @@ var spawn = require('child_process').spawn,
 getOptions();
 
 process.argv.slice(2).forEach(function(arg){
-  var flag = arg.split('=')[0];
+  var flag = arg.split('=', 1)[0].replace(/([a-z])_/g, '$1-');
```

Comment:
If we do this, we will allow only two flags (`--es-staging` and `--use-strict`) to be written with `_`:

1. There is no real reason to make exceptions for these particular two.
2. That will not allow specify other flags with underscores. If you do `node --v8-options`, you will see that v8 lists options with underscores. If `node` allows two separators, I think mocha should too.